### PR TITLE
Update NAMING.md

### PR DESCRIPTION
**Pull request by chandra1011**
Docs (docs-infra) : A small doc change tried. Will it work!

### CODE REVIEW

**Comment by JeanMeche** [comment]
Hi, thank you for improving the docs ! 

For your PR to move forward please signe the CLA and have a look a the contributing rules. The commit message doesn't follow the rules.

**Comment by chandra1011** [comment]
yes, i have signe as CLA

**Comment by JeanMeche** [comment]
For the commit message look at https://github.com/angular/angular/blob/main/CONTRIBUTING.md

**Comment by chandra1011** [comment]
I read the commit message doc ,I think the format is missing ,Tq

**Comment by jessicajaniuk** [comment]
@chandra1011 Please update the commit message to match the required format, and we'll be able to merge this.

File: docs/NAMING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ See: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines
 
 
 Classes:
-  - Example: `Compiler`, `ApplicationMetadata`
+  - Example: `Compiler`and `ApplicationMetadata`
```
Comment (devversion) [inline docs/NAMING.md:9]:
```suggestion
  - Example: `Compiler` and `ApplicationMetadata`
```

**Comment by chandra1011** [comment]
yes ,I have updated commit change according to format .Thank you @angular-team for replying :)

**Comment by JeanMeche** [comment]
@chandra1011 Seems like you haven't pushed anything.

**Comment by jessicajaniuk** [comment]
@chandra1011 Have you pushed that change? The lint CI check is still showing red and there is no new push reported here on github.

**Comment by chandra1011** [comment]
want to make new pull request?

**Review by devversion — CHANGES_REQUESTED** [review]
See my comment above

**Comment by jessicajaniuk** [comment]
Closing this since there's been no updates. Feel free to open a new PR if you'd like to see these changes land.

**Pull request closed without merging**